### PR TITLE
Added Azure CUR integration to the Open Source.

### PR DESCRIPTION
**Pull request by ghost**
## What does this PR change?
* Adds Azure CUR integration into OpenCost, to provide Azure support for the FOCUS spec

## Does this PR relate to any other PRs?
* Open source side of KCM #1507

## How will this PR impact users?
* Provides CloudCost data access for Azure

## Does this PR address any GitHub or Zendesk issues?
* Closes ...

## How was this PR tested?
* locally, with go test

## Does this PR require changes to documentation?
* Updates to show that OpenCost now supports CloudCost data for Azure.

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next Opencost release? If not, why not?
* yes

### CODE REVIEW

File: pkg/cloud/azure/azurestorageintegration.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+				Cost:              abv.NetCost,
+				KubernetesPercent: k8sPtc,
+			},
+		}
+
+		// Check if Item
+		if abv.IsCompute(cc.Properties.Category) {
+			// TODO: Will need to split VMSS for other features
+			computeCCSR.LoadCloudCost(cc)
+		}
```
Comment (Sean-Holcomb) [inline pkg/cloud/azure/azurestorageintegration.go]: Replace with 
```
computeCCSR.LoadCloudCost(cc)
```

File: pkg/cloud/azure/azurestorageintegration.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	"github.com/opencost/opencost/pkg/util/timeutil"
+)
+
+type AzureStorageIntegration struct {
+	AzureStorageBillingParser
+	ConnectionStatus cloud.ConnectionStatus
+}
+
+func (asi *AzureStorageIntegration) GetCloudCost(start, end time.Time) (*kubecost.CloudCostSetRange, error) {
+	computeCCSR, err := kubecost.NewCloudCostSetRange(start, end, timeutil.Day, asi.Key())
```
Comment (Sean-Holcomb) [inline pkg/cloud/azure/azurestorageintegration.go]: Can we rename this variable to just `ccsr`

**Review by Sean-Holcomb — COMMENTED** [review]
Some quick changes then I will approve and merge

File: pkg/cloud/azure/azurestorageintegration.go (hunk 2 above)
Comment (ghost) [reply pkg/cloud/azure/azurestorageintegration.go]: Rename done

**Review by Sean-Holcomb — APPROVED** [review]

**Pull request merged**